models/ResumeModel.ts

```diff
@@ -24,7 +24,7 @@ export class ResumeModel extends BaseEntity {
   public getPublicResume(): PublicResume {
     return {
       uuid: this.uuid,
-      user: this.user.uuid,
+      user: this.user.getPublicProfile(),
```

Comment:
We still want this field to be optional, since `this.user` might not always be defined. So e.g. when a user tries to view their own resumes, which is an operation where the Resume <-> User join is not done, then this line of code will throw an error. We need to do something like:

```ts
const publicResume: PublicResume = {
  uuid: this.uuid,
  isResumeVisible: this.isResumeVisible,
  url: this.url,
  lastUpdated: this.lastUpdated
}
if (this.user) publicResume.user = this.user.getPublicProfile();
return publicResume
```

And `PublicResume` should have `user` as an optional field. Let me know if there's any confusion around this.